### ISSUE TITLE
The restore button is too small

### ISSUE THREAD

**Issue by Shagequi** [nextcloud/android#4848]
Hi,

It's not really a "bug" (nor a feature). And it's pixel perfection.
But the button to restore file in trash is so small and not easy to click/touch.

![Screenshot_20191118-124216_Nextcloud](https://user-images.githubusercontent.com/7050479/69050034-74db5e00-0a01-11ea-8a72-efde0be98919.jpg)

I suggest we replace it by a bigger.

App version : 3.9.0

**Comment by AndyScherzinger** [comment]
cc @nextcloud/designers I agreed with @Shagequi that the icon should be enlarged (as in fit the size of the other 24dp icons). Issue is likely the padding/margin of the icon itself

**Comment by tobiasKaminsky** [comment]
Let us call it a bug, as it prevents user from using the app without problems :-)

**Comment by jancborchardt** [comment]
Yup yup, should have proper clickable size (48*48pt on Android, right?). For these actions in rows, the tappable area of the icon should also extend to the very top and bottom of the row for ideal clickability.

**Comment by AndyScherzinger** [comment]
Size would be 24dp afaik, but the issue is rather the margin within the icon itself

**Comment by jancborchardt** [comment]
Yes, sorry, when I said "clickable size" I mean the padding of the icon. (Margin in CSS at least means _outside_ whitespace, that is non-tappable.)

**Comment by Shagequi** [comment]
Added a new icon from our fork directly

**Comment by ezaquarii** [comment]
@jancborchardt Is there any specific reason why we do not adopt Material Design list components?

1. Those designs have this aspect covered pretty well
2. Users are familiar with this concept as it is used in all Android applications

I can understand that we did this in 4.x as there was no MD back then, but now... hm.

**Comment by jancborchardt** [comment]
> Is there any specific reason why we do not adopt Material Design list components?

@ezaquarii no reason, I assumed we follow Material Design here. :) Definitely something to adopt for this list.

**Comment by ezaquarii** [comment]
@jancborchardt I see... well, we're "material-inspired", while a lot of specification aspects are not used (mostly nitty-gritty details). This might be an accident or deliberate choice, as not everybody likes MD.

https://material.io/components/lists/

Anyway, if we're open to use MD specification, then I'll be mindful of that fact when touching layouts.

**Comment by tobiasKaminsky** [comment]
fixed by https://github.com/nextcloud/android/pull/4865
